Kudu.Core/Infrastructure/PathUtils/PathWindowsUtility.cs

```diff
@@ -148,7 +148,14 @@ internal override string ResolveMSBuild1670Dir()
                 Path.Combine(programFiles, "Microsoft Visual Studio", "2019", "BuildTools",   "MSBuild", "Current", "Bin"), // msbuild tools
                 // above is for public kudu, below is for azure
             };
-            probPaths.Add(Path.Combine(programFiles, "MSBuild-16.8.0", "MSBuild", "Current", "Bin"));
+            string msBuildPath = Path.Combine(programFiles, "MSBuild-16.8.3", "MSBuild", "Current", "Bin");
+            probPaths.Add(msBuildPath);
```

Comment:
Nit: might be cleaner.   if (Directory.Exists()) then add 16.8.3 else add 16.8.0.